docs/source/release_notes/lts/52_0.rst

```diff
@@ -0,0 +1,162 @@
+========
+52.0 LTS
+========
+
+The Avocado team is proud to present another release: Avocado version
+52.0, the second Avocado LTS version.
+
+What's new?
+===========
+
+When compared to the last LTS (v36), the main features introduced by
+this versions are:
+
+- TAP[2] version 12 support.
+- The download of assets in tests now allow for an expiration time.
+- Environment variables can be propagated into tests running on remote
+  systems.
+- The plugin interfaces have been moved into the
+  ``avocado.core.plugin_interfaces`` module.
+- Support for running tests in a Docker container.
+- Introduction of the "Fail Fast" feature (``--failfast on`` option) to
+  the ``run`` command, which interrupts the Job on a first test failure.
+- Special keyword ``latest`` for replaying previous jobs.
+- Support to replay a Job by path (in addition to the Job ID method and
+  the ``latest`` keyword).
+- The introduction of a tool that generated a diff-like report of two
+  jobs.
+- The introduction of a ``rr`` based wrapper.
+- The automatic VM IP detection that kicks in when one uses
+  ``--vm-domain`` without a matching ``--vm-hostname``, now uses a more
+  reliable method (libvirt/qemu-gust-agent query).
+- Set ``LC_ALL=C`` by default on sysinfo collection to simplify avocado
+  diff comparison between different machines.
+- Result plugins system is now pluggable and the results plugins (JSON,
+  XUnit, HTML) were turned into stevedore plugins. They are now listed
+  in the ``avocado plugins`` command.
+- Multiplexer was replaced with Varianter plugging system with defined
+  API to register plugins that generate test variants.
+- Old ``--multiplex`` argument, which used to turn yaml files into
+  variants, is now handled by an optional plugin called ``yaml_to_mux``
+  and the ``--multiplex`` option is being deprecated in favour of
+  the ``--mux-yaml`` option, which behaves the same way.
+- It’s now possible to disable plugins by using the configuration file.
+- Better error handling of the virtual machine plugin (``--vm-domain``
+  and related options).
+- When discovering tests on a directory, the result now is a properly
+  alphabetically ordered list of tests.
+- Plugins can now be setup in Avocado configuration file to run at a
+  specific order.
+- Support for filtering tests by user supplied "tags".
+- Users can now see the tags set on `INSTRUMENTED` test when listing
+  tests with the `-V` (verbose) option.
```

Comment:
Users can see tags of all kind of tests. If you create a loader which will propagate tags, they'll be listed in `avocado list` as well, so I would not stress the `INSTRUMENTED` tests here.